### PR TITLE
feat: added update-clent and IBC update proposals

### PR DESCRIPTION
**Pull request by brianosaurus**
## Description

Adds the update-client proposal handler as well as the IBC update proposal handler. Both are critical for managing IBC connections. Specifically, update-client unfreezes stuck client connections and, as a consequence, stuck tokens.

app.go has 3 changed lines that add these features.

closes: #859 

----

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] added appropriate labels to the PR
- [x] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [x] added a changelog entry to `CHANGELOG.md`
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -918,6 +919,8 @@ func getGovProposalHandlers() []govclient.ProposalHandler {
 		upgradeclient.ProposalHandler,
 		upgradeclient.CancelProposalHandler,
 		leverageclient.ProposalHandler,
+    ibcclientclient.UpdateClientProposalHandler,
+    ibcclientclient.UpgradeProposalHandler,
```
Comment (toteki) [inline app/app.go:923]: gofmt
```suggestion
		leverageclient.ProposalHandler,
 		ibcclientclient.UpdateClientProposalHandler,
 		ibcclientclient.UpgradeProposalHandler,
```

**Comment by adamewozniak** [comment]
@brianosaurus pls also verify your git commits 😉

File: CHANGELOG.md [hunk 2]
```diff
@@ -50,6 +50,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Features
 - [835](https://github.com/umee-network/umee/pull/835) Add miss counter query to oracle cli.
+- [860](https://github.com/umee-network/umee/pull/860) Add IBC upgrade and upgrade-client gov proposals
```
Comment (adamewozniak) [inline CHANGELOG.md:53]:
```suggestion
```

Please move this under the ## [Unreleased] section 😄 details for our cl process are in this file

**Review by adamewozniak — APPROVED** [review]
lgtm! 😄

**Comment by adamewozniak** [comment]
This will go in as a part of the calypso upgrade - #603

**Pull request merged**